src/sentry/static/sentry/app/components/sidebar/index.tsx

```diff
@@ -407,7 +407,11 @@ class Sidebar extends React.Component<Props, State> {
     );
 
     return (
-      <StyledSidebar ref={this.sidebarRef} collapsed={collapsed}>
+      <StyledSidebar
+        ref={this.sidebarRef}
+        collapsed={collapsed}
+        demoMode={ConfigStore.get('demoMode')}
```

Comment:
Can you just inline this into the rendering of the styled component?

```css
  top: ${p => (ConfigStore.get('demoMode') ? p.theme.demo.headerSize : 0)};
```

Reply:
@EvanPurkhiser I could but I guess I'm confused why that would be preferable to doing it in the styled component

Reply:
So that there isn't an extra prop for this thing that will never change value and is essentially a "constant".